src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public abstract class Figure implements DoFigure {

```

Comment:
Ensure that using an abstract class here is appropriate. According to the checklist, one should not use abstract classes to set behavior for classes if an interface would suffice. Consider whether `Figure` should be an abstract class or an interface based on its intended use.